cogs/tomatoes_cog/main.py

```
@@ -0,0 +1,56 @@
+        """
+        user = None
+        
+        if len(ctx.message.mentions) > 0:
+            user = ctx.message.mentions[0]
+        elif ctx.message.reference != None:
+            resolved_message = ctx.message.reference.resolved
+
+            if isinstance(resolved_message, Message):
+                user = resolved_message.author
+
+        if user == None:
```

Ruff (0.15.18)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)